src/main/host/descriptor/socket.h

```diff
@@ -84,7 +84,8 @@ void socket_init(Socket* socket, SocketFunctionTable* vtable,
 void socket_pushInPacket(Socket* socket, Packet* packet);
 void socket_dropPacket(Socket* socket, Packet* packet);
 Packet* socket_pullOutPacket(Socket* socket);
-Packet* socket_peekNextPacket(const Socket* socket);
+Packet* socket_peekNextOutwardPacket(const Socket* socket);
```

Comment:
Nit - maybe just `socket_peekNextOutPacket` and `socket_peekNextInPacket` for consistency with `socket_pullOutPacket`?

Double nit - if we did want to spell it out more, slight preference for `(in|out)Bound` instead of `(in|out)Ward`.

Reply:
I used the shortened form for consistency.